legal.html

```diff
@@ -0,0 +1,73 @@
+---
+permalink: /legal
+layout: front
+---
+
+<header class="masthead text-center">
+  <a class="js-scroll-trigger" href="/#page-top"><img class="logo-apply" src="../img/theplatefund_logo.png"></a>
+</header>
```

Comment:
I think we have this in an include somewhere -- or at least please use one of the updated versions so it works on the translated site.

Reply:
This is related to item #10 on the issue tracker I think.

Reply:
My bad, missed the changes— pushed it into one HTML file that's then only referenced in our sole layout file